versioned/persist/adapter/src/main/java/org/projectnessie/versioned/persist/adapter/spi/AbstractDatabaseAdapter.java

```diff
@@ -186,6 +186,41 @@ public Hash noAncestorHash() {
     return NO_ANCESTOR;
   }
 
+  @Override
+  public Stream<CommitLogEntry> fetchCommitLogEntries(Stream<Hash> hashes) {
+    OP_CONTEXT ctx = borrowConnection();
+
+    BatchSpliterator<Hash, CommitLogEntry> commitFetcher =
+        new BatchSpliterator<>(
+            config.getParentsPerCommit(),
+            hashes,
+            h -> fetchMultipleFromCommitLog(ctx, h, ignore -> null).spliterator(),
```

Comment:
Maybe add constant for `ignore -> null`? It appears to be used in a few places.

Reply:
Better not - no need for constants there (findbugs/checkstyle)

Reply:
fair enough